site/src/components/Navigation/Navigation.styles.js

```diff
@@ -250,17 +250,14 @@ export const Header = styled.header`
     }
     .themeToggle {
       margin: 0 5px;
-      background: ${({ theme }) => theme.body};
```

Comment:
Why were changes made beyond removing `.btn-container`? Removing it alone should resolve the issue, right?

Reply:
I made two additional changes:

1)Removing the background:
When switching the background, adding background: ${({ theme }) => theme.body}; was affecting the background of our SVGs unnecessarily.

2)Removing the border:
This adjustment was made to maintain consistency with the desktop view, where borders were not present, ensuring a cleaner appearance.

Reply:
and in desktop view also there is no switching in background of svg

Reply:
with 
  background: ${({ theme }) => theme.body};

https://github.com/meshery/play/assets/92106389/c68a6b83-5141-47b1-b566-1e6fdab1ae98

without :

https://github.com/meshery/play/assets/92106389/3ced7896-4a74-43e8-bcd4-f86cf5c5e302



with border 
![image](https://github.com/meshery/play/assets/92106389/b2e7fe1a-e107-4160-a8d0-35b653c6c956)
without border 
![image](https://github.com/meshery/play/assets/92106389/3bd574e9-4d00-4af8-ac7f-7242de46a311)

Reply:
@akshansh-modi thanks for the TED Talk on theme toggles and SVG backgrounds! 😄 Appreciate the insights and the videos, though next time, let's aim for a 30-second summary, shall we? Thanks for your contributions!

Reply:
😄 Got it, I'll keep it concise next time.